quartz/i18n/locales/de-DE.ts

```diff
@@ -1,6 +1,21 @@
 import { Translation } from "./definition"
 
 export default {
+  callout: {
+    note: "Hinweis",
+    abstract: "Zusammenfassung",
+    info: "Info",
+    todo: "Zu erledigen",
+    tip: "Tipp",
+    success: "Erfolg",
+    question: "Frage",
+    warning: "Warnung",
```

Comment:
Is warning like the noun 'This is a warning' or more of an exclaimation 'Warning: blablabla' meant?

Reply:
I think it is more of a noun

Reply:
LGTM them

Reply:
Thanks mat. updated accordingly.